daemon/daemon_unix.go

```diff
@@ -86,6 +86,36 @@ func parseSecurityOpt(container *container.Container, config *runconfig.HostConf
 	return err
 }
 
+func getBlkioReadIOpsDevices(config *runconfig.HostConfig) ([]*blkiodev.ThrottleDevice, error) {
+	var BlkioReadIOpsDevice []*blkiodev.ThrottleDevice
+	var stat syscall.Stat_t
+
+	for _, iopsDevice := range config.BlkioDeviceReadIOps {
+		if err := syscall.Stat(iopsDevice.Path, &stat); err != nil {
+			return nil, err
+		}
+		ReadIOpsDevice := blkiodev.NewThrottleDevice(int64(stat.Rdev/256), int64(stat.Rdev%256), iopsDevice.Rate)
+		BlkioReadIOpsDevice = append(BlkioReadIOpsDevice, ReadIOpsDevice)
+	}
+
+	return BlkioReadIOpsDevice, nil
+}
+
+func getBlkioWriteIOpsDevices(config *runconfig.HostConfig) ([]*blkiodev.ThrottleDevice, error) {
```

Comment:
Yes, they look like same.  But I think it's more clear to use two fucntions for readiops and writeiops.
If you insist, I think using 1 validation function is just OK.